language/java/language_test.go

```diff
@@ -113,13 +113,23 @@ func TestLanguageTestFilePath(t *testing.T) {
 		})
 	}
 
-	validate(t, &testCase{
-		Name: "Source file",
+	if !osutil.IsWindows() {
```

Comment:
Just a nitpick but would remove the negation and swap the condition content. When skimming over code it is easier having a `true` case.